### ISSUE TITLE
SS14 By Example uses uid without defining it

### ISSUE THREAD

**Issue by Aeshus** [space-wizards/docs#235]
In the last code snippet in SS14's "Adding a Simple Bikehorn", it uses a `uid` variable which was never defined.

```cs
private void OnUseInHand(Entity<PlaySoundOnUseComponent> ent, ref UseInHandEvent args)
{
    _audio.PlayPvs(ent.Comp.Sound, uid);
}
```

**Comment by Partmedia** [comment]
It should probably be `ent.Owner`

**Comment by Aeshus** [comment]
Done!

**Comment by Partmedia** [comment]
resolved by #236
